build-system/tasks/release/index.js

```diff
@@ -319,10 +318,19 @@ async function fetchAmpSw_(flavorType, tempDir) {
     filter: (path) => path.startsWith('package/dist'),
     strip: 2, // to strip "package/dist/".
   });
-  (await fetch(ampSwTarballUrl)).body.pipe(tarWritableStream);
-  await new Promise((resolve) => {
-    tarWritableStream.on('end', resolve);
-  });
+  const tarballResponse = await fetch(ampSwTarballUrl);
+  if (!tarballResponse.body) {
+    throw new Error(`Failed to fetch ${ampSwTarballUrl}`);
+  }
+  const reader = tarballResponse.body.getReader();
+  while (true) {
+    const {done, value} = await reader.read();
```

Comment:
maybe a dumb question - could `read()` be sync so we don't need `while (true)` ?

Reply:
No, it's an inherently async function. Honestly this whole change annoyed me - in browser implementations `response.body` is a real ReadableStream so we could've left it with `.pipe`, and it's also an async iterator, so we could've done `for await (const chunk of response.body) { ... }`, but Node's implementation of fetch is incompatible :/